doc/format_specification.rst

```diff
@@ -9,6 +9,12 @@ General scope
 
 *  `yaml2PEtab`: Defines a fitting problem of the form `y(t_i) = h(x(t_i), p) + eps_i` with independent Normal or Laplace distributed error terms `eps`. `h` denotes the mapping from system states to observables. PEtab allows to formulate MLE and MAP based fitting problems.
 
+General remarks
+---------------
+
+* All identifiers of states, parameters etc. need to be valid SBML identifiers. Therefore identifiers must consist only of upper and lower case letters, digits and underscores, and must not start with a digit.
```

Comment:
@dilpath native speaker is it "only of" or "of only"?

Reply:
```suggestion
* All identifiers of states, parameters etc. need to be valid SBML identifiers. Therefore identifiers must consist of only upper and lower case letters, digits and underscores, and must not start with a digit.
```